install.sh

```diff
@@ -85,9 +85,6 @@ then
 
 	echo -e "\n[+] go${version}\n"
 
-	https://golang.org/dl/go1.17.6.linux-amd64.tar.gz -o /tmp/go1.17.6.linux-amd64.tar.gz
-	tar -xzf /tmp/go1.17.6.linux-amd64.tar.gz -C /usr/local
-
```

Comment:
I removed these two lines because they don't seem needed. The URL has no command preceding it and thus results in nothing to do for tar.  Go installs without issues with these lines removed.

Reply:
oh! yeah, he do nothing